cli/command/inspect/inspector.go

```diff
@@ -84,6 +87,37 @@ func Inspect(out io.Writer, references []string, tmplStr string, getRef GetRefFu
 	return nil
 }
 
+// All fetches all objects using GetAllFunc and writes the json
+// representation to the output writer.
+func All(out io.Writer, tmplStr string, getAll GetAllFunc) error {
+	inspector, err := NewTemplateInspectorFromString(out, tmplStr)
+	if err != nil {
+		return cli.StatusError{StatusCode: 64, Status: err.Error()}
+	}
+
+	var inspectErr error
+	elements, err := getAll()
+	if err != nil {
+		return err
+	}
+
+	for _, element := range elements {
+		if err := inspector.Inspect(element, nil); err != nil {
+			inspectErr = err
+			break
+		}
+	}
+
+	if err := inspector.Flush(); err != nil {
+		logrus.Errorf("%s\n", err)
+	}
+
+	if inspectErr != nil {
+		return cli.StatusError{StatusCode: 1, Status: inspectErr.Error()}
```

Comment:
Same here - these error codes should be constified and commented somewhere rather than used ad-hoc, or better yet an error constructor should be used to DRY the status code setting - it's not cool if we have to change a `StatusCode` in 30 places!

Reply:
Refactored a little bit, but not sure what to do about the StatusCode, which should be improved probably in a separate PR.